pages.zh/common/accelerate.md

```diff
@@ -0,0 +1,28 @@
+# Accelerate
```

Comment:
Not sure about this, while the library name is Accelerate, we could use it in the page title (in past we have added multiple pages like this with first character capitalized to align with the project name). If this triggers tldr maintainance checks then IG we need to update it to ignore if the first charcter is capitalized (intentionally).

Alternatively, we can discuss and document preference to use full lowercase names in title if the command is fully lowercase too.

Reply:
> The comment itself was more to check if the English page is titlecase or not. Haven't had time to check. AFAIK tldr-maintenance lowercases all titles, so it would not trigger tldr-maintenance.

Yep, the English page uses the same.